src/coreclr/jit/llvm.cpp

```diff
@@ -832,6 +832,30 @@ void Llvm::buildAdd(GenTree* node, Value* op1, Value* op2)
     }
 }
 
+void Llvm::buildDiv(GenTree* node, Value* op1, Value* op2)
+{
+    switch(node->TypeGet())
+    {
+        case TYP_FLOAT:
+        case TYP_DOUBLE:
+            {
+                mapGenTreeToValue(node, _builder.CreateFDiv(op1, op2));
+                break;
+            }
+        default:
+            {
+                if (varTypeIsSigned(node->TypeGet()))
```

Comment:
This will always be `true`. Whether the division is signed or unsigned is decided by the operator: `DIV` is signed, `UDIV` is unsigned.

This is also missing the exceptions: both `DIV` and `UDIV` must throw on a zero divisor, additionally, `DIV` must throw for `INT32/64_MIN / -1`. Ideally this would be done in lowering, but currently we do not have a utility that could correctly split LIR blocks, so it'll have to live in codegen for now.

See `genCodeForDivMod` in `codegenarm64.cpp` for a reference.

(It is also fine to leave the exceptions as `TODO-LLVM` for now)